copy_config.md

```diff
@@ -0,0 +1,78 @@
+# Copy configurations between projects
+
+Copy configurations between projects, from one to one or many other projects.
+
+Configurations to be copied:
+
+- Default layer name
+- Default visible map labels
+- Missing input warning toggle
+- Globals schema
+- Parcels inputs
+    - Schema
+    - Theme
+- Parcels outputs
+    - Theme
+- Parcels aggregation specs
+- Simulations
+    - Order of outputs
+    - Schema
+    - Themes
```

Comment:
I generally don't think we should talk in great detail about what we copy since we will definitely change it and it will be hard to keep in sync.  I think you could mention the important items very generally.